### ISSUE TITLE
Spell check dictionaries fail to download

### ISSUE THREAD

**Issue by longsleep** [iridium-browser/iridium-browser#2]
Try adding a new language (Settings -> Languages) and add a language other than English (eg. German - Deutsch). Then on the right, it offers a button to download spell dictionaries.

Spell check dictionary download failed.
Please check with your network administrator to make sure that the firewall is not blocking downloads from Google servers.

This probably goes to some trk URL. @theurere can you please debug and add the details here.

**Comment by theurere** [comment]
It attempts to fetch:
`https://trk-173.iridiumbrowser.de/cache.pack.google.com/edgedl/chrome/dict/de-de-3-0.bdic`
`https://trk-173.iridiumbrowser.de/cache.pack.google.com/edgedl/chrome/dict/en-us-3-0.bdic`

**Comment by fancycode** [comment]
With 43.1 it tries to fetch this:
```
Attempt at loading traced URL: trk:173:http://cache.pack.google.com/edgedl/chrome/dict/en-us-6-1.bdic
```

**Comment by jengelh** [comment]
The button is just called "Use this language for spell checking". Apart from our trk:173 warning bubble, nothing would otherwise inform the user that a background download takes place, nor that it is Google which is being contacted. Guess I should change the button label.

**Comment by longsleep** [comment]
Good idea. I like it pretty much if we change labels when stuff is loaded from Google on user interaction. Eg. "Load this language from Google".

**Comment by jengelh** [comment]
Makes me wonder if we should just cache it like the safebrowsing list.

**Comment by longsleep** [comment]
+1 sounds good to me.

**Comment by fancycode** [comment]
What is the status of this?

**Comment by fancycode** [comment]
Ping?
